### BIG differences, this is diff of wrong files.
### CLUS "Restart ABORT"
### and CLHS topic selection http://clhs.lisp.se/Body/a_abort.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:restarts:abort.txt	2017-04-27 22:00:06.000000000 +0100
+++ saved-pages-clus/cl:restarts:abort.txt	2017-04-27 22:00:06.000000000 +0100
@@ -1,4 +1,19 @@
-ABORT
-Please select which reference to ABORT you intended
+Restart ABORT

-FunctionRestart
\ No newline at end of file
+Data Arguments Required
+None.
+Description
+The intent of the abort restart is to allow return to the innermost "command level." Implementors are encouraged to make sure that there is always a restart named abort around any user code so that user code can call abort at any time and expect something reasonable to happen; exactly what the reasonable thing is may vary somewhat. Typically, in an interactive listener, the invocation of abort returns to the Lisp reader phase of the Lisp read-eval-print loop, though in some batch or multi-processing situations there may be situations in which having it kill the running process is more appropriate.
+See Also
+\secref\Restarts, {\secref\InterfacesToRestarts}, Function INVOKE-RESTART, Function ABORT
\ No newline at end of file
